validator/rpc/wallet.go

```diff
@@ -165,6 +133,80 @@ func (s *Server) WalletConfig(ctx context.Context, _ *empty.Empty) (*pb.WalletRe
 	}, nil
 }
 
+// RecoverWallet via an API request, allowing a user to recover a derived.
```

Comment:
Can we rephrase the function comment for readability?

Reply:
done